Common.mk

```diff
@@ -411,15 +438,16 @@ validate-checksums: $(BINARY_TARGETS)
 
 # Build helm chart
 .PHONY: helm/build
-
+helm/build: ## Build helm chart
+helm/build: $(if $(or $(wildcard $(PROJECT_ROOT)/helm_patches),$(wildcard $(MAKE_ROOT)/helm_patches)),$(HELM_GIT_PATCH_TARGET),$(HELM_GIT_CHECKOUT_TARGET))
```

Comment:
Move the attribution prereq to this line.

Reply:
I removed it from the helm/push target if that is what was causing the confusion.